server/views/pages/licence/HDC_AP.njk

```diff
@@ -50,11 +50,15 @@
 
    <div id="home-detention-curfew" class="section">
       <p>
-         Your Home Detention Curfew starts on <span class="bold">{{ licence.actualReleaseDate | datetimeToDate or "MISSING - HDC Approved Date/Actual Release Date"}}</span>  and ends on <span class="bold">{{ licence.conditionalReleaseDate | datetimeToDate or "MISSING - Conditional Release Date"}}</span> 
+         Your Home Detention Curfew starts on <span class="bold">{{ licence.homeDetentionCurfewActualDate | datetimeToDate }}</span>  and ends on <span class="bold">{{ licence.conditionalReleaseDate | datetimeToDate }}</span> 
       </p>
    </div>
 
-   {% include "pages/licence/partials/apDates.njk" %}
```

Comment:
Component for `apDates` used `licenceStartDate` instead of the `homeDetentionCurfewActualDate` so cannot use it in this template.